hack/install.sh

```diff
@@ -438,6 +437,10 @@ do_install() {
 			;;
 
 		fedora|centos|redhat|oraclelinux)
+			if [ "${lsb_dist}" = "redhat" ]; then
+				# we use the centos repository for both redhat and centos releases
+				lsb_dist='centos'
+			fi
```

Comment:
@kencochrane yes, I noticed that, but didn't want to make unrelated changes in this PR, can do in a follow up if needed